src/include/migraphx/optimize_module.hpp

```diff
@@ -38,6 +39,7 @@ struct module_pass_manager;
  */
 struct MIGRAPHX_EXPORT optimize_module
 {
+    std::unordered_set<std::string> const_prop_skip_ops = {};
```

Comment:
Can we prefix the variable with the name of the pass? Like `propagate_constant_skip_ops`?